smoke/_init_frr.sh

```
@@ -253,32 +253,27 @@ set_srv6_route() {
-				break
-			fi
-		done
-		[ "$all_respawned" -eq 1 ] && break
-		[ "$SECONDS" -ge 10 ] && \
-			fail "watchfrr did not respawn $* within 10s"
-		sleep 0.1
-	done
+	timeout 10s inotifywait -mqr -e modify --format %f "$frr_run" |
+	while [ "${#pids[@]}" -gt 0 ]; do
+		read -r f
+		unset pids[$f]
```

Shellcheck (0.11.0)

[warning] Quote arguments to unset so they're not glob expanded.

(SC2184)